libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  buildForm,
+  buildMultiField,
+  buildSection,
+  buildSubSection,
+  buildIntroductionField,
+  buildTextField,
+  Form,
+  ApplicationTypes,
+} from '@island.is/application/core'
+import { m } from './messages'
+
+export const ContactInfo: Form = buildForm({
+  id: ApplicationTypes.DOCUMENT_PROVIDER_ONBOARDING,
+  ownerId: 'TODO?',
+  name: m.formName,
+  children: [
+    buildSection({
+      id: 'applicant',
+      name: m.applicantSection,
+      children: [
+        buildMultiField({
+          id: 'applicant',
+          name: m.applicantTitle,
+          description: m.applicantTitle.description,
+          children: [
+            buildTextField({
+              id: 'applicant.name',
+              name: m.applicantName,
+            }),
+            buildTextField({
+              id: 'applicant.email',
+              name: m.applicantEmail,
+            }),
+            buildTextField({
+              id: 'applicant.phoneNumber',
+              name: m.applicantPhoneNumber,
+            }),
+            buildTextField({
+              id: 'applicant.address',
+              name: m.applicantAddress,
+            }),
+            buildTextField({
+              id: 'applicant.zipCode',
+              name: m.applicantZipCode,
+            }),
+          ],
+        }),
+      ],
+    }),
+    buildSection({
+      id: 'contacts',
+      name: m.contacts,
+      children: [
+        buildSubSection({
+          id: 'administrativeContact',
+          name: m.administrativeContactSection,
+          children: [
+            buildMultiField({
+              id: 'administrativeContact',
+              name: m.administrativeContactTitle,
+              description: m.administrativeContactTitle.description,
+              children: [
+                buildTextField({
+                  id: 'administrativeContact.name',
+                  name: m.administrativeContactName,
+                }),
+                buildTextField({
+                  id: 'administrativeContact.email',
+                  name: m.administrativeContactEmail,
+                }),
+                buildTextField({
+                  id: 'administrativeContact.phoneNumber',
+                  name: m.administrativeContactPhoneNumber,
+                }),
+                buildTextField({
+                  id: 'administrativeContact.address',
+                  name: m.administrativeContactAddress,
+                }),
+                buildTextField({
```

Comment:
you can use `width: 'half'` if you want to change the width of fields

Reply:
:+1: